### PR TITLE
New Methylseq release 1.4

### PR DESCRIPTION
**Pull request by alneberg**

### CODE REVIEW

**Comment by ewels** [comment]
Updating the methylseq pipeline to the new v1.4 release. Also fixing up the alias generated by ansible, so that it uses the central nextflow config as expected. Finally removing a bunch of duplicated nextflow config variables for the pipeline that are no longer required.

**Comment by matrulda** [comment]
Looks good to me :+1: But I do not know ansible that well. Maybe @slohse wants to take a look?

**Comment by ewels** [comment]
We will probably do a minor release of 1.4.1 any day now (it’s ready, just needs a PR and release created). Shall we bump this to 1.4.1? One or two minor speed improvements in that hopefully.

**Comment by b97pla** [comment]
should be fine with us

File: roles/methylseq/tasks/main.yml [hunk 1]
```diff
@@ -39,9 +39,13 @@
...
 
 - name: Set nf-core/methylseq Bismark alias
-  lineinfile: dest="{{ ngi_pipeline_conf }}/{{ item.script }}"
-              line="alias methylseq='nextflow run {{ methylseq_dest }}/ -c {{ ngi_pipeline_conf }}/methylseq_{{ item.site }}.config -profile uppmax'"
-              backup=no
+  lineinfile:
+      dest: "{{ ngi_pipeline_conf }}/{{ item.script }}"
+      line: >
+        alias methylseq='nextflow run {{ methylseq_dest }}/
+             -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config
```
Comment (ewels) [inline roles/methylseq/tasks/main.yml]: Did you need a backslash on this line as well..?

File: roles/rnaseq/tasks/main.yml [hunk 2]
```diff
@@ -38,9 +38,9 @@
   lineinfile:
     dest: "{{ ngi_pipeline_conf }}/{{ item.script }}"
     line: >
-          alias rnaseq='nextflow run {{ rnaseq_dest }}/main.nf
-          -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config
-          -c {{ ngi_pipeline_conf }}/rnaseq_{{ item.site }}.config'
+          alias rnaseq='nextflow run {{ rnaseq_dest }}/
+              -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config
```
Comment (ewels) [inline roles/rnaseq/tasks/main.yml]: ..and here

Comment (aanil) [reply roles/rnaseq/tasks/main.yml]: There is a new pull request for them here https://github.com/NationalGenomicsInfrastructure/irma-provision/pull/331

**Pull request merged**